apps/web/src/design-system/icons/general/Star.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
+import styled from '@emotion/styled';
+import { colors } from '../../config';
+
+const StarHolder = styled.div`
+  position: absolute;
+  top: 0;
+  right: 0;
+  transform: translate(50%, -50%);
+  width: 16px;
+  height: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 100%;
+  background: white;
+  border: 1px solid ${colors.gradientMiddle};
+`;
+
+export const Star = (props: React.ComponentPropsWithoutRef<'svg'>) => {
```

Comment:
The star icon, I don't know why but when I export the whole "Primary Provider Icon" from Figma - the icon doesn't have a correct viewport, that's why I just exported the star and made the wrapper circle by hands... I will ask Nikolay to take a look